docs/reference/commandline/build.md

```diff
@@ -300,3 +301,11 @@ like `ENV` values do.
 
 For detailed information on using `ARG` and `ENV` instructions, see the
 [Dockerfile reference](../builder.md).
+
+### Specify isolation technology for container (--isolation)
```

Comment:
Ah, good point.  Thanks @jhowardmsft I was looking for a good def of what is meant by "isolation technology" if you have that it would sweet.

@WeiZhang555 :

Container isolation level > Container isolation technology